web/src/App.less

```diff
@@ -24,6 +24,19 @@ body {
   background: #fbfbff;
 }
 
+h1.ant-typography,
```

Comment:
Had to add this because Ant design won't use the same base color for everything...

Reply:
ok